Makefile

```diff
@@ -41,3 +41,11 @@ $$(BUILDDIR)/$1-%-instance.otf: $$(BUILDDIR)/$1-%-static.otf
 	$$(PYTHON) -m cffsubr -o $$@ $$<
 
 endef
+
+$(DOCSDIR)/preview.pdf: $(DOCSDIR)/preview.tex $(STATICOTFS) | $(BUILDDIR)
+	xelatex --interaction=batchmode -output-directory=$(BUILDDIR) $<
```

Comment:
I use XeTeX for such a sample thing because of the math equations, and sadly there are no other command line tools that can render OpenType math other than *TeX.

Reply:
Yup, hence why I re-instated this rule after having removed under the assumption I could do something else. Most of the example actually renders in SILE's experimental math support branch, but it's not merged yet or anywhere close to ready. I'm tabling this for now until a better solution exists.